iso-configuration/default.nix

```diff
@@ -3,22 +3,23 @@
 {
   imports = [
     ./installer-configuration.nix
-    ../m1-support
+    ../apple-silicon-support
   ];
 
   # include those modules so the user can rebuild the install iso. that's not
   # especially useful at this point, but the user will need the m1-support
   # directory for their own config.
   installer.cloneConfigIncludes = [
     "./installer-configuration.nix"
-    "./m1-support"
+    "./apple-silicon-support"
   ];
 
   # copy the m1-support and installer configs into the iso
   boot.postBootCommands = lib.optionalString config.installer.cloneConfig ''
     if ! [ -e /etc/nixos/m1-support ]; then
+      mkdir -p /etc/nixos/m1-support
       cp ${./installer-configuration.nix} /etc/nixos/installer-configuration.nix
-      cp -r ${../m1-support} /etc/nixos/m1-support
+      cp -rT ${../apple-silicon-support} /etc/nixos/apple-silicon-support
     fi
```

Comment:
This section should be purely a rename, we don't want to add the `mkdir` or change the `cp` flags now that we are back to a single directory.